### ISSUE TITLE
BUG: GeoDataFrame.total_bounds should be a dask array with known shape

### ISSUE THREAD

**Issue by jorisvandenbossche** [geopandas/dask-geopandas#74]
```python
>>> df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_cities"))
>>> ddf = dask_geopandas.from_geopandas(df, npartitions=4)
>>> ddf.total_bounds
dask.array<from_pandas-09ff8835d3deb87b6565b111dcf696cd-total_bounds-agg, shape=(nan,), dtype=float64, chunksize=(nan,), chunktype=numpy.ndarray>
```

This results in a dask array with unknown shape and chunksize, while we know this is always a 1D length 4 array, so I suppose we should be able to tell dask this is the case.

**Comment by jorisvandenbossche** [comment]
Another potential problem with the current `total_bounds` implementation is that it uses `self.reduction(..)` which, according to the docs (https://docs.dask.org/en/latest/dataframe-api.html#dask.dataframe.DataFrame.reduction), is not meant to return arrays. So it might not be that easy to fix the above (unknown shape) with `reduction`.

**Comment by martinfleis** [comment]
This is actually already filed in #4.

**Comment by martinfleis** [comment]
Closed by #75
